handlers/slackCommands/slackCommands_test.go

```
@@ -21,3 +27,90 @@ func TestFindBlankEnvVars(t *testing.T) {
 func TestTimestamp(t *testing.T) {
 	require.Equal(t, "2020-10-29-14-08", timestamp(time.Unix(1603980505, 0)))
 }
+
+func TestFormatMapResponse(t *testing.T) {
+	exclusionStats := map[string]interface{}{
+		"thing1": "hummus",
+		"thing2": 23,
+		"thing3": false,
+	}
+	formattedForSlack := FormatMapResponse(exclusionStats)
+
+	assert.Truef(t, strings.HasPrefix(formattedForSlack, "```"), "Not the right formatting prefix")
+	assert.Contains(t, formattedForSlack, "thing1: hummus")
+	assert.Contains(t, formattedForSlack, "thing2: 23")
+	assert.Contains(t, formattedForSlack, "thing3: false")
+	assert.Truef(t, strings.HasSuffix(formattedForSlack, "\n```"), "Not the right formatting suffix")
+}
+
+func TestGetMeetLink(t *testing.T) {
+	assert.Equal(t, "g.co/meet/pickles-and-hummus", GetMeetLink("pickles and hummus"))
+
+	randomlyGeneratedName := GetMeetLink("")
+	const prefix = "g.co/meet/"
+	assert.Truef(t, strings.HasPrefix(randomlyGeneratedName, prefix), "Starts with prefix")
+
+	// meetName should be three words separated by hyphens like "generously-playful-peanut"
+	meetName := strings.Split(randomlyGeneratedName, prefix)[1]
+	assert.Equal(t, len(strings.Split(meetName, "-")), 3)
+}
+
+func TestBoostCommandsThatAddToTheAzureQueue(t *testing.T) {
```

Unit testing best practice would be to break this into individual tests. At the moment, if one of the earlier checks fails (for a "different test case") then any other failures are hidden away.

For the repetitive cases, I find using t.Run() to help keep things consolidated but by gaining the granularity of the cases

---

yeah i usually try to keep them small, but this one was the closest i could get to an integration test, which we'e been trying to mock out user flow.

i was on the fence about the style at first, but it's kind of nice if you're new to the codebase to find a big ole test that kind of gently walks you through what a user might do